bin/root-transition.sh

```diff
@@ -78,10 +78,13 @@ verify() {
     verify_namespace $n
   done
 
-  if [ -z $NOT_UPDATED ]; then
-    echo "------All Istio mutual TLS keys and certificates match with current root!"
+  if [ -z "$NOT_UPDATED" ]; then
```

Comment:
Can you use `${variable}` pattern in this file?

Reply:
Done.